packages/shared/src/components/modals/PostModal.tsx

```diff
@@ -58,6 +58,33 @@ export function PostModal({
     });
   }, [id]);
 
+  useEffect(() => {
```

Comment:
Love this, but would personally consider moving this to a custom hook, it will make this file a bit less bloated and reusable for the future.

the hook could be something like useKeyboardNavigation and it can accept certain valid keys, en possible return each valid key as a function like onValid{Key}Press

Let me know what you think.

Reply:
I agree